tests/test_text_utils.py

```diff
@@ -83,13 +81,15 @@ def test_alphanumerize(input, output):
 
 
 @pytest.mark.parametrize('input,output', [
-    ['bd', 'boulevard'],
-    ['13e', 'treizieme'],
+    ['bd', ['boulevard']],
```

Comment:
It was the case before, but we used `_synonymize` (before calling `yielder`) to make test easier. But at the end we still return a generator.